src/app/shared/shared.module.ts

```diff
@@ -1,51 +1,53 @@
  /** Angular Imports */
```

Comment:
All unnecessary changes related to indentation should be removed from you commit.